### ISSUE TITLE
Modify tabs with manual activation to support reflow with horizontal scroll buttons

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-practices#3438]
Modify the 
[Example of Tabs with Manual Activation](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-manual/)
so that it supports
[WCAG 1.4.10 Reflow](https://www.w3.org/WAI/WCAG22/Understanding/reflow.html)
regardless of the number of tabs. To support small viewports, enable the tablist to scroll horizontally.
At the left and right ends of the tablist, include scroll helper buttons that are mouse and touch accessible but not keyboard accessible.

  #### Current source files
* [tabs-automatic.html](https://github.com/w3c/aria-practices/blob/b5c817ddc922d0476a8c273dd2e03f55f9070cce/content/patterns/tabs/examples/tabs-automatic.html)
* [tabs.css](https://github.com/w3c/aria-practices/blob/b5c817ddc922d0476a8c273dd2e03f55f9070cce/content/patterns/tabs/examples/css/tabs.css)

Note:
[tabs.css](https://github.com/w3c/aria-practices/blob/b5c817ddc922d0476a8c273dd2e03f55f9070cce/content/patterns/tabs/examples/css/tabs.css)
is used by both
[tabs-manual.html](https://github.com/w3c/aria-practices/blob/b5c817ddc922d0476a8c273dd2e03f55f9070cce/content/patterns/tabs/examples/tabs-manual.html)
and by 
[tabs-automatic.html](https://github.com/w3c/aria-practices/blob/b5c817ddc922d0476a8c273dd2e03f55f9070cce/content/patterns/tabs/examples/tabs-automatic.html).
Going forward, these two examples of tabs will no longer be able to share the same CSS because they will each illustrate a slightly different approach to supporting reflow.
So, this issue requires copying tabs.css into a new tabs-manual.css and changing tabs-manual.html to use tabs-manual.css in lieu of tabs.css.

#### [WIP] Requirements
* Make the example tablist a horizontal scroll container.
* When tabs have scrolled out of view  in either direction, show a scroll button adjacent to the end of the tablist that indicates the availability of more tabs in that direction and that will scroll the next set of tabs into view when activated.
* the scroll buttons should be outside the tablist container, accessible to touch screen readers, and should not be included in the keyboard Tab sequence.
* Add more tabs to make it easier for readers to experiment with how the example supports reflow.
* Document the 1.4.10 support approach and accessibility aspects of design decisions in the accessibility features section.
